scripts/server

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env python
+
+import os
+from BaseHTTPServer import HTTPServer
+from SimpleHTTPServer import SimpleHTTPRequestHandler
+
+PORT = 54633
+PATH_TO_PROJECT = 'src/GeositeFramework/'
+
+class CustomRootHTTPServer(HTTPServer):
+    def __init__(self, base_path, *args, **kwargs):
+        HTTPServer.__init__(self, *args, **kwargs)
+        self.RequestHandlerClass.base_path = base_path
+
+class CustomRootHTTPRequestHandler(SimpleHTTPRequestHandler):
+    def translate_path(self, path):
+        words = filter(None, path.split('/'))
+        modified_path = self.base_path
+        for word in words:
+            modified_path = os.path.join(modified_path, word)
+        return modified_path
+
+def serve():
+    server = CustomRootHTTPServer(PATH_TO_PROJECT, ('', PORT), CustomRootHTTPRequestHandler)
+    server.serve_forever()
```

Comment:
Minor, but consider logging a message that the server has started.

Reply:
I like it